src/ipc.c

```diff
@@ -90,6 +90,8 @@ umf_result_t umfGetIPCHandle(const void *ptr, umf_ipc_handle_t *umfIPCHandle,
         return ret;
     }
 
+    // ipcData->handle_id is filled by tracking provider
+    ipcData->base = allocInfo.base;
```

Comment:
so do we expect allocInfo.base != NULL? if so, please check this

Reply:
`allocInfo.base` cannot be `NULL`. The `base` is a key in the memory tracker. If `umfMemoryTrackerGetAllocInfo` is able to find the memory region than `base` is not `NULL`. Otherwise the `umfMemoryTrackerGetAllocInfo` returns an error that we check on L62.